### PR TITLE
Apex-Assignment 2 Commit

### PR DESCRIPTION
**Pull request by Sarita-Rathore-1306**
Created TrainingSessionManager Class with the following methods-
1.usecaseOfEnum
2.createRecordsOfTrainingSession
3.readRecordsFromTrainingSession
4.updateRecordsFromTrainingSession
5.deleteRecordsFromTrainingSession
6.fetchRelatedRecordsOfAttendeesFromTrainingSession 7.getAllRecordsOfTrainingSessionWithTheirRelatedAttendeesRecords 8.addAttendeesToTrainingSession

### CODE REVIEW

File: src/Apex-Assignment 2/classes/TrainingSessionManager.cls [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+ Return         : void
+ Argument       : Id,String
+ Revision_Log   : V1.0
+ */
+public static void updateRecordsFromTrainingSession(ID id, String location){
+  for(Training_Session__c trainingSessionToBeUpdated:[SELECT Id, Name, Location__c FROM Training_Session__c]){
+   try{
+    if(trainingSessionToBeUpdated.Id == id){
+      trainingSessionToBeUpdated.Location__c=location;
+      update trainingSessionToBeUpdated;
```
Comment (Preeti-fexle-training) [inline src/Apex-Assignment 2/classes/TrainingSessionManager.cls:76]: update the query and also we never perform DML inside for loop.

**Review by Preeti-fexle-training — CHANGES_REQUESTED** [review]

File: src/Apex-Assignment 2/classes/TrainingSessionManager.cls [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+}
+/*
+ Purpose        : This Method will read records from Training Session.
+ Return         : void
+ Argument       : Id 
+ Revision_Log   : V1.0
+ */
+public static void readRecordsFromTrainingSession(ID id){
+  List<Training_Session__c> readRecords = new List<Training_Session__c>();
+  for(Training_Session__c trainingSessionsToBeReaded:[SELECT Id,Name FROM Training_Session__c]){
```
[the remark this answers is not in the document]
Comment (Sarita-Rathore-1306) [reply src/Apex-Assignment 2/classes/TrainingSessionManager.cls]: Ok mam,I have updated the code.

File: src/Apex-Assignment 2/classes/TrainingSessionManager.cls [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  }
+  
+}
+/*
+ Purpose        : This Method will read records from Training Session.
+ Return         : void
+ Argument       : Id 
+ Revision_Log   : V1.0
+ */
+public static void readRecordsFromTrainingSession(ID id){
```
[the remark this answers is not in the document]
Comment (Sarita-Rathore-1306) [reply src/Apex-Assignment 2/classes/TrainingSessionManager.cls]: Mam  I have updated the code. Pease check.

File: src/Apex-Assignment 2/classes/TrainingSessionManager.cls (hunk 1 above)
Comment (Sarita-Rathore-1306) [reply src/Apex-Assignment 2/classes/TrainingSessionManager.cls]: Updated the Code, Please Check.

**Review by Preeti-fexle-training — APPROVED** [review]

**Pull request merged**